Formula/r/rtabmap.rb

```diff
@@ -25,13 +25,35 @@ class Rtabmap < Formula
   end
 
   depends_on "cmake" => [:build, :test]
+
   depends_on "g2o"
+  depends_on "libpng"
   depends_on "librealsense"
+  depends_on "lz4"
   depends_on "octomap"
   depends_on "opencv"
   depends_on "pcl"
   depends_on "pdal"
 
+  on_macos do
+    depends_on "boost"
+    depends_on "flann"
+    depends_on "glew"
+    depends_on "libomp"
+    depends_on "libpcap"
+    depends_on "qhull"
+    depends_on "qt"
+    depends_on "sqlite"
+    depends_on "vtk"
+  end
+
+  on_linux do
```

Comment:
are these right? https://github.com/Homebrew/homebrew-core/actions/runs/9993807198/job/27630461929#step:4:611

```
Indirect dependencies with linkage:
    qt
    sqlite
    vtk
    zlib
```

Reply:
looks like I got something wrong, just fixed it.